dotCMS/src/main/java/com/dotcms/rest/api/v1/page/PageForm.java

```diff
@@ -0,0 +1,102 @@
+package com.dotcms.rest.api.v1.page;
+
+import java.io.IOException;
+import java.util.Map;
+
+import com.dotcms.contenttype.transform.JsonTransformer;
+import com.dotcms.repackage.com.fasterxml.jackson.annotation.JsonProperty;
+import com.dotcms.repackage.com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.dotcms.repackage.javax.validation.constraints.NotNull;
+import com.dotcms.rest.exception.BadRequestException;
+import com.dotmarketing.portlets.templates.design.bean.TemplateLayout;
+import com.dotmarketing.util.json.JSONException;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.dotmarketing.util.json.JSONException'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)